config/environments/production.rb

```diff
@@ -0,0 +1,120 @@
+require "active_support/core_ext/integer/time"
```

Comment:
Whats this file about?

Reply:
This file lets you configure how your application will work in production environments.